src/async-wrap.h

```diff
@@ -8,6 +8,8 @@
 
 namespace node {
 
+#define ASYNC_ID_OFFSET 0xA1C
```

Comment:
Is there a deterministic way to generate a class id? that's pretty much how it's done everywhere else in core.